pac4j-oauth/src/main/java/org/pac4j/oauth/client/FacebookClient.java

```diff
@@ -81,6 +83,8 @@
     protected boolean requiresExtendedToken = false;
     
     protected StateApi20 api20;
+
+    protected String appsecretProof = null;
```

Comment:
Oh, I see. Okay. I have several other pressing tasks today, but may have something better for review in 8 to 12 hours.